### ISSUE TITLE
How to use custom models for the video classification example

### ISSUE THREAD

**Issue by kihapper** [ml5js/ml5-examples#20]
I am trying to build a web-application based on the [video classification](https://ml5js.github.io/docs/video-classification-example.html) example.
Following the google tensorflow-for-poets tutorial [here]( https://codelabs.developers.google.com/codelabs/tensorflow-for-poets/index.html?index=..%2F..%2Findex#3
) I made a re-trained custom model based on [mobilenet_v2_1.0_224.pb]. 
It has 5 classes to differentiate and would really help if I can run that on the browser.
Now I have the [custom_model.pb]  and the [custom_model.lite] version following[ these steps ](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets-2-tflite/#2)to optimize it for mobile usage.

The question is...
**Is there a way to implement custom models in the video classification example?**

I tried to dig into the original code of ml5.js but it was difficult to grasp where the models were loaded.
It seems now it only supports the default MobileNet/SqueezeNet.

Thanks for reading and lots of love to ML5.js :)

**Comment by shiffman** [comment]
@kihapper For now I would suggest looking at the new `FeatureExtractor` example which allows you train new labels on top of a pre-trained model ('MobileNet'). 

https://github.com/ml5js/ml5-examples/tree/master/p5js/FeatureExtractor_Image_Classification

If this is sufficient, please go ahead and close! We are tracking this new feature for training from scratch here: https://github.com/ml5js/ml5-library/issues/121

**Comment by cvalenzuela** [comment]
Closing this since ml5js/ml5-library#121 is keeping track of this! feel free to reopen if necessary
